api_tests/lib/wallets/ethereum.ts

```diff
@@ -107,17 +106,9 @@ export class EthereumWallet implements Wallet {
         return this.inner.getAccount();
     }
 
-    public async deployErc20TokenContract(
-        projectRoot: string
-    ): Promise<string> {
+    public async deployErc20TokenContract(): Promise<string> {
         const data =
```

Comment:
Could you actually put it at the end of the file in a const to help readability?

Reply:
Do you have word wrap turned on or what is the issue?

Reply:
I just think that a giant variable is better at the end of a test file.